blogs:{{KMeans_images}}:proposal.md

```diff
@@ -0,0 +1,50 @@
+# TEMPLATE
+
+## :fire: Do not edit this file - copy the template and create your own file.
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+KMeans Clustering
```

Comment:
The title needs to be more attention grabbing. 
AN idea could be "Extracting Features from Images through K Means Clustering"